homeassistant/components/litterrobot/sensor.py

```diff
@@ -91,6 +93,7 @@ def icon(self) -> str | None:
             name="Last Seen",
             device_class=SensorDeviceClass.TIMESTAMP,
             entity_category=EntityCategory.DIAGNOSTIC,
+            state_class=SensorStateClass.TOTAL_INCREASING,
```

Comment:
Does it make sense for a timestamp entity to have a state class?

Reply:
I don't think this change belongs.

Reply:
That is indeed incorrect. Please remove this state class.

Reply:
Removed